avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/Avro2TF.scala

```diff
@@ -88,6 +88,8 @@ object Avro2TF {
    */
   case class SparseVector(indices: Seq[Long], values: Seq[Float])
 
+  case class RankedSparseVector(indices: Seq[Seq[Long]], values: Seq[Float])
```

Comment:
I feel the name is somewhat misleading. We don't rank sparseVector, but only put an additional rank. It's more like a SparseVectorList, which can be used for Ranking use cases.

Reply:
Perhaps we call it `ListwiseSparseVector`, since it's meant to accommodate list-wise examples in a ranking context?

Also, I'd suggest adding a scaladoc comment so that, no matter what name we choose, we have a clear explanation for this class.

Reply:
Sounds good. I created another pull request and renamed it as MultiDimensionSchema (https://github.com/linkedin/Avro2TF/pull/62) I will close this pull request